coopAndLastFM2.js

```diff
@@ -137,50 +137,32 @@ var app = {
 			$("#searchResults").resizable();
 		});
 		
+		$("#save").click(function() {
+
+			saveCanvas('test', 'jpg');
+
+		});
 
 	},
 	
 
 };
 
 // global variable to hold onto the image
-<<<<<<< HEAD
-//var theImage;
-=======
 
```

Comment:
These are all merge conflicts, some things you have to pay attention to when you collaborate on a repo. Don't worry too much about this right now